holidata/holidays/es-ES.py

```diff
@@ -3,22 +3,58 @@
 
 from .holidays import Locale
 
+"""
+Information taken from government websites around 2020-06
+    https://administracion.gob.es/pag_Home/atencionCiudadana/calendarios/laboral.html
+    http://www.seg-social.es/wps/portal/wss/internet/CalendarioLaboral
+    2020: https://www.boe.es/diario_boe/txt.php?id=BOE-A-2019-14552
+    2019: https://www.boe.es/diario_boe/txt.php?id=BOE-A-2018-14369
+
+Regional governments
+    [AN] https://www.juntadeandalucia.es/temas/trabajar/relaciones/calendario.html
+
+Also those sites for some information
+    https://es.wikipedia.org/wiki/Calendario_laboral
+"""
 
 class es_ES(Locale):
     """
     01-01: [NF] Año Nuevo
-    01-06: [NRF] Día de los Reyes
+    01-06: [NRF] Epifanía del Señor
+    02-28: [AN] [F] Día de Andalucía
+    03-01: [IB] [F] Día de las Illes Balears
+    03-13: [ML] [F] Estatuto de Autonomía de la Ciudad de Melilla
+    03-19: [GA,MC,NC,PV,VC] [RF] San José
+    04-23: [AR] [RF] San Jorge / Día de Aragón
+    04-23: [CL] [F] Fiesta de Castilla y León
     05-01: [NF] Fiesta del Trabajo
+    05-02: [MD] [F] Fiesta de la Comunidad de Madrid
+    05-17: [GA] [F] Día de las Letras Gallegas
+    05-30: [CN] [F] Día de Canarias
+    05-31: [CM] [F] Día de Castilla-La Mancha
+    06-09: [MC] [F] Día de la Región de Murcia
+    06-09: [RI] [F] Día de La Rioja
+    06-24: [CT,VC] [RF] San Juan
+    07-25: [GA,PV] [RF] Santiago Apóstol / Día Nacional de Galicia
```

Comment:
You are right I just copied like this because official government documentation put it that way.
07-25 is the religious holiday of Santiago Apóstol but only in Galicia is the region main day because of the importance of the Santiago (Saint Jacques) has in this region. Its capital is named _Santiago de Compostela_. Reference: https://en.wikipedia.org/wiki/Santiago_de_Compostela. And it is the meeting end of a very famous pilgrims' ways in Europe: https://en.wikipedia.org/wiki/Camino_de_Santiago. 

I would suggest:
```
07-25: [GA] [RF] Santiago Apóstol / Día Nacional de Galicia
07-25: [PV] [RF] Santiago Apóstol

```
What do you think? Keep it or modify it like above?

Reply:
Or maybe for simplicity, just keep the _santiago Apóstol_ part as a Galician would know already this is the Galician national day.
`07-25: [GA,PV] [RF] Santiago Apóstol`

Reply:
Anyway I think the important part is the religious holiday for both regions.